catalog/transformer/catalog-transformer-csv-queryresponse/src/test/java/ddf/catalog/transformer/csv/CsvMetacardTransformerTest.java

```diff
@@ -67,7 +67,7 @@
   public void setUp() {
     this.transformer = new CsvMetacardTransformer();
     this.arguments = new HashMap<>();
-    arguments.put("columnOrder", ImmutableList.of("stringAtt", "intAtt", "doubleAtt"));
```

Comment:
Something to consider is using the `columnOrder` constant here as well in case the constant value ever changed (however maybe unlikely). I wouldn't hold up the PR for it. Would the test just fail in that case?

Reply:
@shaundmorris yes, good catch. the other prs have already been merged, otherwise i'd make the change.